docs/spec.md

```diff
@@ -495,6 +498,26 @@ Unsubscribe from [connection graph updates](#connection-graph-update). Only supp
 }
 ```
 
+### Fetch asset
+
+Fetch an asset from the server. Only supported if the server previously declared that it has the `assets` [capability](#server-info).
+
+#### Fields
+
+- `op`: string `"fetchAsset"`
+- `assetURI`: string, uniform resource identifier to locate a single asset
```

Comment:
> However, I think it could be worth calling out that different servers may interpret URIs in different ways

That's exactly how URIs work, and one of the distinguishing features from the more specific subset of URIs known as URLs. `tel:+1-816-555-1212` is a valid URI but it doesn't say anything about how to "fetch" that resource, or if "fetch" is even a valid verb for that resource in the context of a given server.

I support the requirement that assets are identified by a URI.